src/leiningen/i18n.clj

```diff
@@ -79,6 +82,20 @@
   (let [root (:root project)]
     (io/as-file (apply path-join (cons root rest)))))
 
+(defn update-scripts-with-pot-name
+  "CI script task needs to know the name of the POT file to be modified and
+   committed. This will insert the correct name of the POT in the /bin files."
+  [project]
+  (let [dest (path-join (dev-resources-dir project) "i18n" "bin" "update-pot.sh")
+        pot-script (io/resource "leiningen/i18n/bin/update-pot.sh")]
+    (spit (io/as-file dest)
+          (clojure.string/replace-first
```

Comment:
I know this was in there already, but we should add `clojure.string` to the requires and alias it, rather than sidestepping the require process by referring to the fully qualified namespace and hoping that a dependency loads that namespace first.